tests/coverage/branch.coverage

```diff
@@ -0,0 +1,120 @@
+   LL|       |#![feature(coverage_attribute)]
+   LL|       |// edition: 2021
+   LL|       |// compile-flags: -Zunstable-options -Cinstrument-coverage=branch
+   LL|       |
+   LL|       |macro_rules! do_not_merge_with_signature {
+   LL|       |    () => { if true {} else {} };
+  ------------------
+  |  Branch (LL:16): [True: 15, False: 0]
+  ------------------
+  |  Branch (LL:16): [True: 15, False: 0]
+  ------------------
+  |  Branch (LL:16): [True: 15, False: 0]
+  ------------------
+  |  Branch (LL:16): [True: 15, False: 0]
+  ------------------
+   LL|       |}
+   LL|       |
+   LL|     15|fn if_not(cond: bool) {
+   LL|     15|    do_not_merge_with_signature!();
+   LL|       |
+   LL|       |    if
+   LL|       |        !
+   LL|     15|        cond
+  ------------------
+  |  Branch (LL:9): [True: 10, False: 5]
+  ------------------
```

Comment:
Two problems here:
- The span doesn't include `!`.
- The true/false counts are reversed, since they're tracking `cond` instead of `!cond`.